pkg/provider/provider_test.go

```diff
@@ -4,12 +4,64 @@ import (
 	"testing"
 
 	"github.com/chanzuckerberg/terraform-provider-snowflake/pkg/provider"
+	"github.com/hashicorp/terraform/helper/schema"
+	_ "github.com/snowflakedb/gosnowflake"
```

Comment:
just curious, what is being init'd here?

Reply:
The sql driver for snowflake. https://github.com/snowflakedb/gosnowflake/blob/master/driver.go#L141